### ISSUE TITLE
user verificationStatus always as 'GUEST'

### ISSUE THREAD

**Issue by alvarocardenasc** [jovotech/jovo-cli#322]
## I'm submitting a...

- [x] Bug report
- [ ] Feature request
- [ ] Documentation issue or request
- [x] Other... Please describe:

## Current Behavior

I cant save persistent data in Google Action because the user verificationStatus is always as GUEST

### Error log
```
"user": {
    "locale": "en-US",
    "params": {},
    "accountLinkingStatus": "NOT_LINKED",
    "verificationStatus": "GUEST",
    "packageEntitlements": [],
    "gaiamint": "",
    "permissions": [],
    "lastSeenTime": "2022-02-21T23:26:04Z"
  }
```
What I'm missing?

## Your Environment

* Jovo CLI version used: 4.1.3
* Operating System:  10.15.5

**Comment by aswetlow** [comment]
Hey @alvarocardenasc 

Is it a GSuite Account you're testing with? 

![image](https://user-images.githubusercontent.com/14978292/155091099-e75f6b80-14dd-4193-9a05-70955e387d28.png)


Here's more context: https://community.jovo.tech/t/how-is-jovo-affected-by-deprecation-of-google-userid-request-field/547/18

**Comment by alvarocardenasc** [comment]
yes. Would it work with a personal account?

**Comment by alvarocardenasc** [comment]
yes. But I tried with a personal gmail account and get the same result (Alpha tester):
this.$googleAssistant?.$user.isVerified() -> false (with personal @gmail.com account)

**Comment by aswetlow** [comment]
yes, it should. Have you tried to enable voice matching?
